src/framework/core/js/FluidView.js

```diff
@@ -448,6 +448,17 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     /**
+     * Returns the document to which an element belongs, or the element itself if it is already a document
+     *
+     * @param @param {jQuery||Element} element the element to return the document for
+     * @return {Document} dokkument the document in which it is to be found
+     */
```

Comment:
Is it useful to add documentation about this at http://docs.fluidproject.org/infusion/development/ViewAPI.html